04week/nasa/css/style.css

```diff
@@ -0,0 +1,50 @@
+.galaxy{
+  background-image: url("../img/nasa-background.jpg");
+  background-repeat:no-repeat;
+  height:450px;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+}
+
+body{
+  width:850px;
+}
+header{
+font-weight: bold;
+font-size:20px;
+margin-bottom: 20px;
+}
+
+
+
+.info{
+  display: flex;
+  height:324px;
+  justify-content:space-around;
+  align-items: baseline;
+  flex-direction: row;
+  background-color: #E6E6E6;
+  margin-top: 20px;
+  margin-left: 2px;
+}
+
+.office-info{
+width:270px;
+  height:320px;
```

Comment:
You already provided the height to the parent (.info) and this is making it bleed out of it's parent.
![image](https://user-images.githubusercontent.com/8717281/28447018-fa9b967a-6d93-11e7-8d93-6eee1de5f52d.png)
Same goes for `.contact-us`